src/toast/toast.js

```diff
@@ -37,11 +37,12 @@ class Toast extends React.Component<ToastPropsT, ToastPrivateStateT> {
   };
 
   autoHideTimeout: ?TimeoutID;
+  animateInTimer: ?TimeoutID;
   animateOutCompleteTimer: ?TimeoutID;
 
   state = {
-    isAnimating: false,
-    isHidden: true,
+    isVisible: false,
+    isRendered: true,
```

Comment:
These names felt a little bit clearer. `isRendered` controls whether or not it's actually mounted in the DOM, `isVisible` controls whether it's shown.

Reply:
`isHidden` and `isAnimating` were introduced in order to handle height animation in addition to the opacity so the rest of notifications shifted in a smooth way when a single notification from a list of stacked notifications is dismissed

Reply:
Still seems to be slightly confusing since we set `isRendered` to true as an initial state that doen't mean component is mounted.

Reply:
> isHidden and isAnimating were introduced in order to handle height animation in addition to the opacity so the rest of notifications shifted in a smooth way when a single notification from a list of stacked notifications is dismissed

Maybe I'm missing something, but height animation didn't work at all so I sort of just nuked that functionality (see later comment)

> Still seems to be slightly confusing since we set isRendered to true as an initial state that doesn't mean component is mounted.

Yeah that's sort of why I chose the name `isRendered` instead of `isMounted`, it's meant to signify that render method should return something (not `null`).